src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -132,10 +132,17 @@ fn check_fixture_decorator(checker: &mut Checker, func_name: &str, decorator: &E
 
                 if let Some(scope_keyword) = scope_keyword {
                     if keyword_is_literal(scope_keyword, "function") {
-                        checker.diagnostics.push(Diagnostic::new(
+                        let mut diagnostic = Diagnostic::new(
                             violations::ExtraneousScopeFunction,
                             Range::from_located(scope_keyword),
-                        ));
+                        );
+                        if checker.patch(diagnostic.kind.rule()) {
+                            diagnostic.amend(Fix::deletion(
+                                scope_keyword.location,
+                                scope_keyword.end_location.unwrap(),
+                            ));
+                        }
```

Comment:
I think we need to be a bit more careful here. E.g. with `@pytest.fixture(scope="function",)`, would this leave the trailing comma and thus lead to a syntax error? What if this is the last argument, and we don't remove the preceding comma?

Trying to find an example of where we handle this... I guess `remove_class_def_base`? But maybe not the best implementation.